code/modules/mob/living/living.dm

```diff
@@ -951,11 +952,11 @@
 		if(AM == pulling)// Are we trying to pull something we are already pulling? Then just stop here, no need to continue.
 			return
 		stop_pulling()
-		if(AM.pulledby)
-			visible_message("<span class='danger'>[src] has pulled [AM] from [AM.pulledby]'s grip.</span>")
-			AM.pulledby.stop_pulling() //an object can't be pulled by two mobs at once.
+		for(var/mob/M in AM.pulledby)
+			visible_message("<span class='danger'>[src] has pulled [AM] from [M]'s grip.</span>")
+			M.stop_pulling() //an object can't be pulled by two mobs at once.
```

Comment:
This looks like it might get a bit spammy when a mob is pulled out of a bunch of mobs' hands at once.